packages/runtime/lib/management-api.js

```diff
@@ -26,6 +26,8 @@ async function createManagementApi (configManager, runtimeApiClient) {
     return packageJson
   }
 
+  app.register(require('@fastify/websocket'))
```

Comment:
I think this might need to be set in package.json?

Reply:
it's already there